Source/Provers/SMTLib/SMTLibProcessTheoremProver.cs

```diff
@@ -644,6 +645,23 @@ public override void Reset(VCExpressionGenerator gen)
       }
     }
 
+    private void RecoverIfProverCrashedAfterReset()
+    {
+      resetCount += 1;
+      if (Process.HasProverDied() is { } e)
```

Comment:
I think I would find `is Exception e` easier to read than `is { } e`

Reply:
That's was the IDE semi-auto correction (it underlined in yellow `ProverDiedException e` when I wrote it), which was annoying.
Thanks to your suggestion, I'm writing `Exception e` and it does not complain anymore and indeed it's easier to read.